index.js

```diff
@@ -335,8 +335,12 @@ morgan.token('user-agent', function getUserAgentToken (req) {
  */
 
 morgan.token('req', function getRequestToken (req, res, field) {
+  if (!field) {
+    throw new Error('req is missing an argument')
+  }
+
   // get header
-  var header = req.headers[field.toLowerCase()]
+  var header = req.headers[field.toLowerCase()];
```

Comment:
The code style in use for this project doesn't use semicolons and this line doesn't need to be changed in this PR anyhow.

```suggestion
  var header = req.headers[field.toLowerCase()]
```